File: app/src/main/java/com/taxi/friend/drivers/models/ResponseWrapper.java
package com.taxi.friend.drivers.models;

import java.util.List;

public class ResponseWrapper<T> {
    private List<T> result;

    public ResponseWrapper() {
    }

    public ResponseWrapper(List<T> result) {
        this.result = result;
    }

    public List<T> getResult() {
        return result;
    }

    public void setResult(List<T> result) {
        this.result = result;
    }
}

File: app/src/main/java/com/taxi/friend/drivers/models/ResponseWrapper.kt
package com.taxi.friend.drivers.models


import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonInclude

@JsonIgnoreProperties(ignoreUnknown = true)
@JsonInclude(JsonInclude.Include.USE_DEFAULTS)

class ResponseWrapper<T>(
        var errors: ErrorMessage? = null,
        var result: T? = null
)
